cerebrium/advanced-functionality/processing-functions.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "Processing Functions"
+---
+
+In many cases, you may want to process the data in your single or multi-model flow before returning it to the next flow stage or to the user. To do this, you can supply a both a pre-processing and a post-processing function to your flow, `pre` and `post` respectively. 
+The pre-processing will be applied to the input of the model, while the post-processing function will be applied to the output of your model before it is returned to the next flow stage. 
+You will **need** to implement both of these functions if you wish to use binary files along with your imput data.
+
+In particular:
+- The pre-processing function should take 1-2 arguments **in order**: the input to your model and any input files as binary data. It should return a single value, which is the pre-processed input to your model. It is <b>crucial</b> that the output of your pre-processing function is the same shape and type as the input of the model in the flow stage.
```

Comment:
I would also add here to check out persistant memory to send objects across stages